.travis.yml

```diff
@@ -76,7 +87,12 @@ install:
   - psql -d qiita_test -c 'select * from qiita.study;'
   - labcontrol patch
 script:
-  - nosetests --with-doctest --with-coverage -v --cover-package=labcontrol
+  # test javascript unit-testing using qunit and chrom{e|ium} using the
+  # node-qunit-puppeteer script and a trivial qunit-based test in a new
+  # directory called ctests.
+  # - node-qunit-puppeteer labcontrol/gui/test/index.html 10000  "--allow-file-access-from-files --no-sandbox"
```

Comment:
I think this comment, and the commented-out line, are somewhat out-of-date.  Could you update?

Reply:
Done, thanks for the catch!